builder/digitalocean/builder.go

```diff
@@ -41,6 +41,10 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 		return nil, warnings, errs
 	}
 
+	if len(warnings) > 0 {
+		return nil, warnings, nil
```

Comment:
Here, I think that we should probably always return everything without an if statement 🤔

Reply:
Super nice catch !